file_paths.md

```diff
@@ -204,6 +217,20 @@ user or group.
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.cfg                     # pxp-agent configuration file
+            modules.d *                       # directory where pxp-agent modules are stored
+                puppet.cfg *                  # ../../var/modules/puppet configuration
+        var *
+            log
+                pxp-agent.log                 # enabled by default
+            modules *
```

Comment:
This `modules` binary directory doesn't seem correct to me given it's inside of a user's data directory.

On *nix, we have a dir of `/etc/puppetlabs/pxp-agent/modules.d` for stored modules, and `/opt/puppetlabs/pxp-agent/modules/` for their binaries.

But on Windows we're proposing everything for Windows be placed under the Windows equivalent of `/etc/puppetlabs/pxp-agent` (`C:\ProgramData\PuppetLabs\pxp-agent`).  I assume that `modules.d` is separate from `modules` for a reason, so shouldn't we preserve that on Windows?

I don't think MCO is quite a good parallel to look at either, because I don't think MCO separated plugins from their binaries?  At least in the spec its `/opt/puppetlabs/mcollective/plugins` vs `C:\ProgramData\PuppetLabs\mcollective\plugins` (which is a little weird).